pkg/rbac/userData.go

```diff
@@ -235,13 +259,27 @@ func (user *UserDataCache) getNamespacedResources(cache *Cache, ctx context.Cont
 
 		}
 	}
+	klog.V(7).Infof("User %s has access to these namespace scoped res: %+v \n", uid, user.userData.NsResources)
+	klog.V(7).Infof("User %s has access to these ManagedClusters: %+v \n", uid, user.userData.ManagedClusters)
 
 	user.nsrUpdatedAt = time.Now()
 	user.clustersUpdatedAt = time.Now()
 
 	return user, user.nsrErr
 }
 
+//SSRR has resources that are clusterscoped too
+func (shared *SharedData) isClusterScoped(kindPlural, apigroup string) bool {
```

Comment:
🤔 I wonder if it is possible to create a cluster-scoped CR with the same group and name as a namespaced CR.
This would definitely be an edge scenario.